frontend/components/common/PageBreadcrumb.tsx

```diff
@@ -37,15 +39,22 @@ export function PageBreadcrumb({ items, ...otherProps }: PageBreadcrumbProps) {
       <Breadcrumb
          spacing={1}
          separator={
-            <Icon
-               as={HiChevronRight}
+            <Flex
                display={{
                   base: 'none',
                   md: 'initial',
                }}
-               color="gray.300"
-               mt="1"
-            />
+            >
+               <FontAwesomeIcon
+                  icon={faChevronRight}
+                  color={theme.colors.gray[400]}
+                  style={{
+                     height: '10px',
+                     display: 'flex',
+                     marginTop: '4px',
+                  }}
+               />
```

Comment:
I'd suggest we create a wrapper around `FontAwesomeIcon` using https://chakra-ui.com/docs/styled-system/chakra-factory#chakra-factory-function and use that wrapper everywhere, so that it supports the Chakra magic props and theming.

Reply:
Good proposal. I'm working on it

Reply:
Updated